.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,82 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
```

Comment:
This assumes that the issue is in the serverpod_cli and that it's related to a command, which may not always be the case?
Perhaps it would useful to customize the form such that it provides the means to submit issues for the `serverpod_cli`, `server`, `client`, `tests`, `examples`

Reply:
In addition, would it make sense to add as well the running mode? 

- Development
- Staging
- Production